src/content/products/deepsparse/enterprise.mdx

```diff
@@ -300,7 +300,7 @@ Use Case: A workload that might benefit from the elastic scheduler is one in whi
 
 ### License
 
-The Community Edition of the project's binary containing the DeepSparse Engine is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
-Example files and scripts included in this repository are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
+The Community Edition of the project's binary containing DeepSparse is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
```

Comment:
This needs to say Neural Magic DeepSparse Community License.

Reply:
We're just waiting on the second reviewer and this will be updated per the edits of this PR.